DbAdapterTestUtilities/Stubs/StubFieldValue.h

```diff
@@ -17,17 +17,12 @@ namespace systelab::db::test_utility {
 		StubFieldValue(const db::IFieldValue&);
 		StubFieldValue(const StubFieldValue&);
 		StubFieldValue(const std::string& name);
-		StubFieldValue(const std::string& name, int value);
-		StubFieldValue(const std::string& name, bool value);
-		StubFieldValue(const std::string& name, double value);
-		explicit StubFieldValue(const std::string& name, const std::string& value);
-		StubFieldValue(const std::string& name, const std::chrono::system_clock::time_point& value);
 
-		StubFieldValue(const std::string& name, const std::optional<int>& value);
-		StubFieldValue(const std::string& name, const std::optional<bool>& value);
-		StubFieldValue(const std::string& name, const std::optional<double>& value);
-		StubFieldValue(const std::string& name, const std::optional<std::string>& value);
-		StubFieldValue(const std::string& name, const std::optional<std::chrono::system_clock::time_point>& value);
+		explicit StubFieldValue(const std::string& name, const std::optional<int>& value);
```

Comment:
The problem with this is that probably then you'll be force to especifically set an optional on the value side of things, which if the value was not meant to be optional is an abuse.
If you don't want to have so many constructors, maybe we can try either a template approach or the usage of std::variant (with proper order of types, like bool at the end) .

Reply:
Makes sense, but due to time constraints I'll restore the initial constructors but moving the explicit qualifier to the bool constructor.